pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -512,6 +521,67 @@ func (og *operationGenerator) GenerateMountVolumeFunc(
 	}, volumePlugin.GetPluginName(), nil
 }
 
+func (og *operationGenerator) checkForResize(volumeToMount VolumeToMount, devicePath string, mounter mount.Interface) (bool, error) {
```

Comment:
Name `checkForResize` suggests that it does only some checks. It actually performs full resize.

Reply:
function renamed